bundles/org.openhab.binding.dominoswiss/src/main/java/org/openhab/binding/dominoswiss/internal/BlindHandler.java

```diff
@@ -0,0 +1,176 @@
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        logger.debug("Blind got command: {} and ChannelUID: {} ", command.toFullString(),
+                channelUID.getIdWithoutGroup());
+        dominoswissHandler = (EGateHandler) getBridge().getHandler();
+        if (dominoswissHandler == null) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_UNINITIALIZED, "EGate not available");
+            logger.debug("Blind thing {} has no server configured, ignoring command: {}", getThing().getUID(), command);
+            return;
+        }
+
+        // Some of the code below is not designed to handle REFRESH
+        if (command == RefreshType.REFRESH) {
+            return;
+        }
+        switch (channelUID.getIdWithoutGroup()) {
+            case CHANNEL_PULSEUP:
+                if (command instanceof Number) {
+                    dominoswissHandler.pulseUp(id);
+                }
+                break;
+            case CHANNEL_PULSEDOWN:
+                if (command instanceof Number) {
+                    dominoswissHandler.pulseDown(id);
+                }
+                break;
+            case CHANNEL_CONTINOUSUP:
+                if (command instanceof Number) {
+                    dominoswissHandler.continuousUp(id);
+                }
+                break;
+            case CHANNEL_CONTINOUSDOWN:
+                if (command instanceof Number) {
+                    dominoswissHandler.continuousDown(id);
+                }
+                break;
+            case CHANNEL_STOP:
+                if (command instanceof Number) {
+                    dominoswissHandler.stop(id);
+                }
+                break;
+            case UP:
+                if (command instanceof Number) {
+                    dominoswissHandler.continuousUp(id);
+                }
+                break;
+            case DOWN:
+                if (command instanceof Number) {
+                    dominoswissHandler.continuousDown(id);
+                }
+                break;
+            case SHUTTER:
+                if (command.toFullString() == DOWN) {
+                    dominoswissHandler.continuousDown(id);
+                } else if (command.toFullString() == UP) {
+                    dominoswissHandler.continuousUp(id);
+                } else if (command.toFullString() == CHANNEL_STOP) {
+                    dominoswissHandler.stop(id);
+                } else {
+                    logger.debug("Blind got command but nothing executed: {}  and ChannelUID: {}",
+                            command.toFullString(), channelUID.getIdWithoutGroup());
+                }
+
+            case TILTDOWN:
+                if (command instanceof Number) {
+                    try {
+                        dominoswissHandler.tiltDown(id);
+                    } catch (InterruptedException e) {
+                        logger.error("EGate tiltDown error: {} ", e.toString());
```

Comment:
An `InterruptedException` is thrown when openHAB is stopped. As this is a normal operation, this shouldn't be logged. Please check all.